types/inquirer/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Inquirer.js 6.x
+// Type definitions for Inquirer.js 7.0
```

Comment:
This number is supposed to be purely just the npm version of the JavaScript library these typings are for. Since the type defs themselves have breaking changes, perhaps the best solution would be to wait until Inquirer actually ships a new major version 😬

Reply:
It's a pity that it's not possible to change the version of the generated package somehow.
The major version has not been changed for about a year now so I really don't think there's going to be another major version in the near future.

Reply:
@jedmao
If the major version is not being increased, people using `@types/inquirer` might have to refactor parts of their code due to breaking changes.

Reply:
If all the pre-existing tests pass, I don't see why this would be the case. What's breaking? Sounds like an assumption.

Reply:
I renamed `Question<T>` to `DistinctQuestion<T>` in favor of adding a question-interface which represents the base of all question-interfaces.

That's the only breaking change so far.

Reply:
To @andrewbranch's point, I'm not sure you understand what this version represents. If I'm using an older version of Inquirer, say 6.3.1 and I `npm install @types/inquirer@6` then I would expect to get the latest types for Inquirer@6, which means less than 7. If you bump this "types" version to 7, this creates two problems:
1. `npm install @types/inquirer@6` isn't getting the latest types for the Inquirer's major version of 6, as they would expect.
1. When Inquirer v7 is actually published, the types for v7 are actually for v6, which is terribly confusing and wrong.

In short, this version is _never_ supposed to reference a non-existing or future version of any package. It's not a shame or a pity that we can't "change this somehow" because we're not supposed to be breaking current implementations and preserving existing tests is a great way to ensure that.

Reply:
Aah I get you.
I just updated the version in the header to "6.5" which is the actual current minor-version of inquirer.